chore: use single, not double clicks for SizedPlot

jrenee42 opened the pull request:
add singleClick to interactionHandlers, and make the SizedPlot respond to single; not doubleclicks

hoorayimhelping commented:
@jrenee42 did you add the updated package.json?

also, semantic nitpick, i think this is probably a `feat`, not a `chore` since we're adding capability

jrenee42 commented:
> @jrenee42 did you add the updated package.json?
> 
> also, semantic nitpick, i think this is probably a `feat`, not a `chore` since we're adding capability

good catch.  just added package.json; and changed chore: to feat: for the first commit too.

hoorayimhelping reviewed (approved):
Awesome!
